pom.xml

```diff
@@ -122,6 +122,8 @@
 
         <extra.parameters />
 
+        <helm.debug>false</helm.debug>
```

Comment:
I am leaning towards making this `true` as otherwise `helm` is very quiet and it will be potentially also useful output in case customers are going to reach out to Support.

Reply:
I agree, this makes sense. I've changed the default.